Traceback (most recent call last):
  File "/home/masumoto/anaconda3/envs/py36_tf114/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/masumoto/anaconda3/envs/py36_tf114/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/masumoto/coper/CoPER_ConvE/qa_cpg/run_cpg.py", line 54, in <module>
    elif use_parameter_lookup:
NameError: name 'use_parameter_lookup' is not defined
